doc/release-notes.md

```diff
@@ -63,6 +63,14 @@ RPC changes
 
 - The `fundrawtransaction` rpc will reject the previously deprecated `reserveChangeKey` option.
 
+- The `redeemScript` for each of the `prevtxs` given to `signrawtransaction` can now optionally be an array
+  of scripts, so both the redeemScript and witnessScript for P2SH-P2WSH addresses can be passed in.
+
+- The `listunspent` output will now return an array of scripts as the `redeemScript` in the case of P2SH-P2WSH,
+  for compatibility with the `signrawtransaction` change above. In other cases, `listunspent` will continue to return
+  the only `redeemScript` as a string, as before.
```

Comment:
Is there a reason to break the API here? Why not add a new key for the witnessScript?

Reply:
The API only breaks in the case of P2SH-P2WSH which was not supported at all before anyway. Adding a new key for witnessScript to listunspent would make its output incompatible with the signrawtransaction input which expects a list of scripts in this case